README.md

```diff
@@ -6,7 +6,7 @@
 Material for the
 [GMT](https://www.generic-mapping-tools.org)/[PyGMT](https://www.pygmt.org)
 pre-conference workshop (PREWS9) at the
-[AGU24 Annual Meeting](https://www.agu.org/annual-meeting).
+[AGU24 Annual Meeting](https://www.agu.org/annual-meeting-2024).
```

Comment:
Did you check if there are other occurrences of the AGU website we have to update? Sofar I did not find more.

Reply:
I did a search and am pretty sure there are only these 2 instances. Also ran the entire repo through [`lychee`](https://github.com/lycheeverse/lychee) and didn't find any other broken links.